EndlessClient/Dialogs/Factories/ItemTransferDialogFactory.cs

```diff
@@ -44,7 +46,19 @@ public ItemTransferDialog CreateItemTransferDialog(string itemName, ItemTransfer
                 transferType,
                 totalAmount,
                 message);
-            dlg.DialogClosing += (_, _) => _sfxPlayer.PlaySfx(SoundEffectID.DialogButtonClick);
+
+            dlg.DialogClosing += (_, _) =>
```

Comment:
`ItemTransferDialogFactory` shouldn't have/need any changes, the sound effect should be added to the controller where the `warningMsg` dialog is created and shown.

This class should stay generic to creating a dialog for transferring items somewhere, not have special logic for gold drop.

Reply:
Sure, I'll do that 👍